api/src/modules/geo-regions/geo-region.repository.ts

```diff
@@ -68,6 +70,9 @@ export class GeoRegionRepository extends Repository<GeoRegion> {
       .values({
         name: () => `hashtext(:arg1)`,
         theGeom: () => `ST_GeomFromText(:arg2, 4326)`,
+        h3Flat: () => `array(SELECT h3_geo_to_h3(ST_GeomFromText(:arg2), 6))`,
+        h3FlatLength: () =>
+          `cardinality(array(SELECT h3_geo_to_h3(ST_GeomFromText(:arg2), 6)))`,
         h3Compact: () =>
           `array( SELECT (h3_compact(array(SELECT h3_geo_to_h3(ST_GeomFromText(:arg2), 6)))))`,
```

Comment:
Again, I am aware this is out if this PR's scope, but one question. Does this query have a guard against a conflicting 'name' field, as the above query has?